ballerina/tests/mock_service.bal

```diff
@@ -0,0 +1,468 @@
+// Copyright (c) 2024, WSO2 LLC. (http://www.wso2.com).
+//
+// WSO2 LLC. licenses this file to you under the Apache License,
+// Version 2.0 (the "License"); you may not use this file except
+// in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+import ballerina/http;
+import ballerina/log;
+
+listener http:Listener httpListener = new (9090);
+
+http:Service mockService = service object {
+    # set_profile_photo
+    #
+    # + return - set_profile_photo response 
+    resource function post account/set_profile_photo(@http:Payload SetProfilePhotoArg payload) returns SetProfilePhotoResult|http:Response {
+        return {
+            "profile_photo_url": "https://dl-web.dropbox.com/account_photo/get/dbaphid%3AAAHWGmIXV3sUuOmBfTz0wPsiqHUpBWvv3ZA?vers=1556069330102&size=128x128"
+        };
+    }
+
+    # create_folder
+    #
+    # + return - create_folder response 
+    resource function post files/create_folder_v2(@http:Payload CreateFolderArg payload) returns FolderMetadataResponse|http:Response {
```

Comment:
```suggestion
    resource function post files/create_folder_v2(CreateFolderArg payload) returns FolderMetadataResponse|http:Response {
```

Reply:
Since this is `post` method we don't need to explicitly mentioned requestBody payload by adding annotation.

Reply:
Updated all the `post` methods.